library/pk_wrap.h

```diff
@@ -31,6 +31,10 @@
 #include "psa/crypto.h"
 #endif /* MBEDTLS_PSA_CRYPTO_C */
 
+#if defined(MBEDTLS_PK_CAN_ECDSA_VERIFY) || defined(MBEDTLS_PK_CAN_ECDSA_SIGN)
```

Comment:
* In a legacy configuration: `MBEDTLS_ECDSA_C` provides both public-key and private-key support.
* In a PSA configuration: `PSA_WANT_KEY_TYPE_ECC_KEY_PAIR` implies `PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY` and we have no desire to change that.

So `MBEDTLS_PK_CAN_ECDSA_SOME` is effectively synonymous with `MBEDTLS_PK_CAN_ECDSA_VERIFY`.